Homework2/src/problem3/Main.java

```diff
@@ -0,0 +1,15 @@
+package problem3;
+
+public class Main {
+    public static void main(String[] args) {
+        User user1 = new User("Math", "math33@gmail.com");
+        User user2 = new User("Kim", "kim@gmail.com");
+        Message message1 = new Message("Description", "User2");
```

Comment:
e ok, doar ca nu exista in implementarea ta nici o relatie intre User si Message: in Message ai fi putut folosi in loc de String author un User author si cand ai printa mesajele ai face: message.getAuthor().getName()